.gitignore

```diff
@@ -12,6 +12,7 @@
 */_build/
 /_sources/
 /.tox
+charts/
```

Comment:
Did you mean to add this? we don't have a charts directory.

Reply:
Yes for the skaffold dev setup to ignore that folder else shows alot of noise in your commits tab in VSCode